src/nist.py

```diff
@@ -68,85 +68,123 @@ def get_NIST_ids():
-            '12': {'16': '43.79', '18': '43.24','20': '42.89', '22': '42.72', '24': '42.70', '26': '42.80'},
-            '11': {'16': '61.49', '18': '60.70','20': '60.21', '22': '59.97', '24': '60.00', '26': '60.17'},
-            '10': {'16': '87.47', '18': '86.41','20': '85.75', '22': '85.03', '24': '85.01', '26': '85.28'},
-            '9': {'16': '122.99', '18': '121.79','20': '121.08', '22': '120.80', '24': '120.90', '26': '121.34'},
-            '8': {'16': '177.68', '18': '175.94','20': '174.95', '22': '174.59', '24': '174.78', '26': '175.48'},
-            '7': {'16': '243.77', '18': '241.84','20': '240.86', '22': '240.75', '24': '241.31', '26': '242.45'},
-            '6': {'16': '343.00', '18': '341.53','20': '341.58', '22': '342.58', '24': '344.23', '26': '346.67'},
-            '5': {'16': '483.91', '18': '482.91','20': '484.97', '22': '486.92', '24': '490.24', '26': '494.55'},
-            '4': {'16': '675.07', '18': '686.88','20': '690.08', '22': '695.01', '24': '701.06', '26': '709.48'},
-            '3': {'16': '950.71', '18': '963.56','20': '987.27', '22': '1000.81', '24': '1015.70', '26': '1030.78'},
-            '2': {'16': '1274.07', '18': '1317.71','20': '1330.16', '22': '1355.29', '24': '1367.79', '26': '1395.94'},
-            '1': {'16': '1766.68', '18': '1830.34','20': '1883.97', '22': '1937.34', '24': '1987.50', '26': '2066.95'}
-            };
     
-        ##Get keys and values of dictionary to construct a 2D array##
-        #Get dictionary keys as lists
-        sphereID = list(phantom_v2.keys());
-        temperature = list(phantom_v2['1'].keys());
+def temperature_correction(input_temperature,serial_number,interpolation='quadratic'):
+    """
+    NIST phantom temperaure correction tool
     
-        #Define array (data) to store data for interpolation
-        data = np.empty([len(temperature),len(sphereID)+1]);
-        #Fill data array
-        row = 0;
-        for i in temperature:
-            row = row + 1;
-            data[row-1,0] = int(i);
-            for j in sphereID:
-                T1_value = float(phantom_v2[j][i]);
-                data[row-1,int(j)] = T1_value;
-
-        ##Code for Temperature Correction: Interpolation##
-        #If input_temperature is a scalar:
-        #Validate input_temperature
-        validTemperature = lambda input_temperature: np.isscalar(input_temperature);
-        if validTemperature(input_temperature) == True:
-            input2array = input_temperature;
-            input_temperature = np.arange(1);
-            input_temperature[0] = input2array;
+    INPUT ARGUMENTS:
+        input_temperature - Temperature (°C) at which the phantom measurements were taken
+        serial_number - Phantom serial number
+        interpolation - 'quadratic' (default), 'cubic' and 'cubic-spline':
+            'quadratic' - A low order polynomial was used to fit a log-log representation of the data
```

Comment:
This line isn't clear to me, why is the option explained in the past tense "was". What will the option do (e.g. in the future) if I select it?